.devcontainer/devcontainer.json

```diff
@@ -9,7 +9,28 @@
       "**/.git/subtree-cache/**": true,
       "target/**": true
     },
-    "dprint.path": "/home/vscode/.dprint/bin/dprint"
+    "[jsonc]": {
```

Comment:
This file seems to be a combination of `.vscode/settings.json` and `.vscode/extensions.json`.

It might be a good idea to add a deno task to generate it.

thoughts?

Reply:
Sounds like another thing we'd have to maintain. The configuration duplication is a bit frustrating... but let's hard-code it for now. Seems we can update the extension lists / get rid of Rust-related extensions such as `serayuzgur.crates`.